src/main/java/com/rotilho/jnano/commons/Hashes.java

```diff
@@ -17,11 +13,15 @@ private Hashes() {
     }
 
     public static byte[] digest(int digestSize, byte[]... byteArrays) {
-        requireNonNull(byteArrays, "Byte Arrays can't be null");
+    	if (byteArrays == null) {
+    		throw new NullPointerException("Byte Arrays can't be null");
+    	}
```

Comment:
This is the equivalent of requireNonNull essentially, but could disregard this since the null pointer exception would occur eventually regardless if `byteArrays` is null